src/components/PageList.vue

```diff
@@ -285,8 +333,13 @@ export default {
 <style lang="scss" scoped>
 
 .scroller {
-	// NC header bar 50px; page list header bar 52px; landing page 48px; page trash 76px
-	height: calc(100vh - 50px - 52px - 48px - 76px);
+	// NC header bar 50px; page list header bar 52px; landing page 48px; page trash 76px NcAppNavigationCaption 78px divided by 2 for multiple scrollers
+	max-height: calc((100vh - 50px - 52px - 48px - 76px - 78px * 2) / 2);
```

Comment:
Nitpick to use a css var where we have one:

```suggestion
	max-height: calc((100vh - var(--header-height) - 52px - 48px - 76px - 78px * 2) / 2);
```